polkadot/README.md

```diff
@@ -1,13 +1,264 @@
+                    paritytech/ci-linux:production cargo build --release
+sudo chown -R $(id -u):$(id -g) target/
+```
+
+If you want to reproduce other steps of CI process you can use the following 
+[guide](https://github.com/paritytech/scripts#gitlab-ci-for-building-docker-images).
+
+## Networks
+
+This repo supports runtimes for Polkadot, Kusama, and Westend.
+
+### Connect to Polkadot Mainnet
+
+Connect to the global Polkadot Mainnet network by running:
+
+```bash
+./target/release/polkadot --chain=polkadot
+```
+
+You can see your node on [telemetry] (set a custom name with `--name "my custom name"`).
+
+[telemetry]: https://telemetry.polkadot.io/#list/Polkadot
+
+### Connect to the "Kusama" Canary Network
+
+Connect to the global Kusama canary network by running:
+
+```bash
+./target/release/polkadot --chain=kusama
+```
+
+You can see your node on [telemetry] (set a custom name with `--name "my custom name"`).
+
+[telemetry]: https://telemetry.polkadot.io/#list/Kusama
+
+### Connect to the Westend Testnet
+
+Connect to the global Westend testnet by running:
+
+```bash
+./target/release/polkadot --chain=westend
+```
+
+You can see your node on [telemetry] (set a custom name with `--name "my custom name"`).
+
+[telemetry]: https://telemetry.polkadot.io/#list/Westend
+
+### Obtaining DOTs
+
+If you want to do anything on Polkadot, Kusama, or Westend, then you'll need to get an account and
+some DOT, KSM, or WND tokens, respectively. See the
+[claims instructions](https://claims.polkadot.network/) for Polkadot if you have DOTs to claim. For
+Westend's WND tokens, see the faucet
+[instructions](https://wiki.polkadot.network/docs/learn-DOT#getting-westies) on the Wiki.
+
+## Hacking on Polkadot
+
+If you'd actually like to hack on Polkadot, you can grab the source code and build it. Ensure you have
+Rust and the support software installed. This script will install or update Rust and install the
+required dependencies (this may take up to 30 minutes on Mac machines):
+
+```bash
+curl https://getsubstrate.io -sSf | bash -s -- --fast
+```
+
+Then, grab the Polkadot source code:
+
+```bash
+git clone https://github.com/paritytech/polkadot-sdk.git
+cd polkadot
+```
+
+Then build the code. You will need to build in release mode (`--release`) to start a network. Only
+use debug mode for development (faster compile times for development and testing).
+
+```bash
+./scripts/init.sh   # Install WebAssembly. Update Rust
+cargo build # Builds all native code
+```
+
+You can run the tests if you like:
+
+```bash
+cargo test --workspace --release
+```
+
+You can start a development chain with:
+
+```bash
+cargo build
+cargo run -- --dev
+```
+
+Detailed logs may be shown by running the node with the following environment variables set:
+
+```bash
+RUST_LOG=debug RUST_BACKTRACE=1 cargo run -- --dev
+```
+
+### Development
+
+You can run a simple single-node development "network" on your machine by running:
+
+```bash
+polkadot --dev
```

Comment:
there're no instructions to build the polkadot binary and put it on your path in this section

```suggestion
cargo run --bin polkadot --release -- --dev
```